packages/backend-app-api/src/alpha/featureDiscoveryServiceFactory.ts

```diff
@@ -84,13 +107,31 @@ class PackageDiscoveryService implements FeatureDiscoveryService {
       if (!LOADED_PACKAGE_ROLES.includes(depPkg?.backstage?.role ?? '')) {
         continue;
       }
-      const depModule = require(require.resolve(name, { paths: [packageDir] }));
-      for (const exportValue of Object.values(depModule)) {
-        if (isBackendFeature(exportValue)) {
-          features.push(exportValue);
-        }
-        if (isBackendFeatureFactory(exportValue)) {
-          features.push(exportValue());
+
+      const exportedModulePaths = [
+        require.resolve(name, {
+          paths: [packageDir],
+        }),
+      ];
+
+      // Find modules exported as alpha
+      try {
+        exportedModulePaths.push(
+          require.resolve(`${name}/alpha`, { paths: [packageDir] }),
+        );
+      } catch (e) {
+        /* No alpha exports so ignore  */
+      }
```

Comment:
Open to suggestions on how best to detect and handle alpha exports.
This is currently a hack around linting not allowing empty catch blocks

Reply:
This is the way.

Although I'd prolly go for ` catch { /* ignore */ }` 😁

Reply:
Fixed in [ee7566f](https://github.com/backstage/backstage/pull/19674/commits/ee7566fc4fee62e953016e38a7857dc357664c0b)